workshops/vue/minis/mini6.md

```diff
@@ -127,7 +129,7 @@ After that, the Quiz component has to be registered within **App.vue**, which is
 
 ```javascript
 components: {
-    Quiz
+  Quiz;
```

Comment:
remove this lint-provided semi-colon, I don't think that's the correct syntax for this